### ISSUE TITLE
Skip errors when `#[serde(skip)]` applied

### ISSUE THREAD

**Issue by oscartbeaumont** [oscartbeaumont/specta#153]
Eg.
```rust
pub struct Demo {
   a: String,
   #[skip(skip)]
   b: Box<dyn std::error::Error> // <- Will error on not supporting `dyn` even though field is skipped
}
```

This is a regression when we moved to runtime handling of skip.

**Comment by oscartbeaumont** [comment]
This applied to enum variants and fields. We should not process them and instead just return `None` instead of `Some(DataType)`

**Comment by IgnisDa** [comment]
@oscartbeaumont I am not sure about `skip(skip)` but I do have `serde(skip)` and specta is still trying to include it in the `Type` macro.

**Comment by oscartbeaumont** [comment]
Yeah meant to write `serde(skip)`/`specta(skip)`, in the original issue whoops. This should be fixed by #154 so maybe try using `main`. Will hopefully do a release in the next week or so with a fix.
